src/Common/Common.Api/Common.Api.csproj

```diff
@@ -8,8 +8,8 @@
   </PropertyGroup>
   
   <ItemGroup>
-    <PackageReference Include="Microsoft.PowerBI.Api" Version="2.0.121817003" />
-    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
+    <PackageReference Include="Microsoft.PowerBI.Api" Version="2.5.1905001" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
```

Comment:
This will be interesting as Newtonsoft.Json causes us issues in the past as PowerShell sessions can only load one version of Newtonsoft.Json which is used by many other PowerShell cmdlets. This could also impact Cloud Shell so we could try to roll out but it may have to be backed out. Is there a reason why the version was upgraded?

Reply:
Removed this dependecny and suppressed the warning